From: ciceron <Stephane.Carrez@gmail.com>
Date: Sat, 28 Jan 2023 14:27:04 +0100
Subject: Update to follow Ada Debian policy

- use gnatcoll and xmlezout provided by Ada Debian packages
  (must provide a xml_ez_out.gpr project because a git dependent
  project is using it instead of xmlezout.gpr)
- don't link alr in static mode
---
 alire_common.gpr | 4 ++--
 alr_env.gpr      | 4 +---
 xml_ez_out.gpr   | 5 +++++
 3 files changed, 8 insertions(+), 5 deletions(-)
 create mode 100644 xml_ez_out.gpr

diff --git a/alire_common.gpr b/alire_common.gpr
index 8228cb1..e228180 100644
--- a/alire_common.gpr
+++ b/alire_common.gpr
@@ -68,11 +68,11 @@ abstract project Alire_Common is
    end Compiler;
 
    package Builder is
-      for Switches ("Ada") use ("-s", "-j0", "-g");
+      for Switches ("Ada") use ("-j0", "-g");
    end Builder;
 
    package Binder is
-      for Switches ("Ada") use ("-Es", "-g", "-static");
+      for Switches ("Ada") use ("-Es", "-g");
    end Binder;
 
    package Ide is
diff --git a/alr_env.gpr b/alr_env.gpr
index edca82f..5aeee2d 100644
--- a/alr_env.gpr
+++ b/alr_env.gpr
@@ -8,7 +8,6 @@ aggregate project Alr_Env is
                          "deps/ajunitgen",
                          "deps/ansi",
                          "deps/clic",
-                         "deps/gnatcoll-slim",
                          "deps/minirest",
                          "deps/optional",
                          "deps/semantic_versioning",
@@ -17,8 +16,7 @@ aggregate project Alr_Env is
                          "deps/spdx",
                          "deps/stopwatch",
                          "deps/toml_slicer",
-                         "deps/uri-ada",
-                         "deps/xmlezout"
+                         "deps/uri-ada"
                         );
 
    for Project_Files use ("alr.gpr");
diff --git a/xml_ez_out.gpr b/xml_ez_out.gpr
new file mode 100644
index 0000000..c6bb0d3
--- /dev/null
+++ b/xml_ez_out.gpr
@@ -0,0 +1,5 @@
+with "xmlezout";
+
+project Xml_Ez_Out is
+   for Source_Dirs use ();
+end Xml_Ez_Out;
